soc/openhwgroup/cva6/cv64a6/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+# RISCV64 OpenHW Group cva6 configuration options
+
+# Shared properties
+config SOC_CV64A6
+	bool
+	select RISCV
+	select RISCV_PRIVILEGED
+	select ATOMIC_OPERATIONS_BUILTIN
+	select RISCV_GP
+	select RISCV_HAS_PLIC
+	select USE_SWITCH_SUPPORTED
+	select USE_SWITCH
+	select SCHED_IPI_SUPPORTED
+	select 64BIT
+	select RISCV_ISA_RV64I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
+	select RISCV_ISA_EXT_C
+	select RISCV_ISA_EXT_F
+	select RISCV_ISA_EXT_D
```

Comment:
Could you use aggregate extensions here?

Reply:
The only aggregate extension currently defined is G, which is IMAFD + fence + csr. As you correctly pointed out above, the non-FPU version should not enable F and D.

Reply:
In that case - yeah, let's leave it as it currently is